easybuild/easyconfigs/p/PyTorch/PyTorch-1.7.1-foss-2020b.eb

```diff
@@ -0,0 +1,308 @@
+name = 'PyTorch'
+version = '1.7.1'
+
+homepage = 'https://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
```

Comment:
Should we really say "GPU acceleration" in the description?

Reply:
It's the standard description taken from https://github.com/pytorch/pytorch...

I understand your point though, people may assume they're using a GPU-accelerated installation.

Then again, is using different descriptions in different easyconfigs maintainable?

Reply:
well, alright, i don't think users really read these anyway. An alternative would be to switch for both of them, with the description from pytorch.org "An open source machine learning framework that accelerates the path from research prototyping to production deployment.". Or not.